pkg/kv/kvserver/kvflowcontrol/replica_rac2/admission.go

```diff
@@ -192,116 +188,3 @@ func (p *lowPriOverrideState) getEffectivePriority(
 	}
 	return pri
 }
-
-// waitingForAdmissionState records the indices of individual entries that are
-// waiting for admission in the AC queues. These are added after emerging as
-// MsgStorageAppend in handleRaftReady, hence we can expect monotonicity of
-// indices within a leader term, and if the term changes, we can assume that
-// non-monotonicity means raft log indices being overwritten and the log being
-// truncated. These assumptions simplify the processing of add, since it can
-// wipe out higher indices.
-//
-// We've chosen to track individual indices instead of the latest index, to
```

Comment:
Might want to copy some of this comment into `LogTracker`.

Reply:
> Might want to copy some of this comment into LogTracker.

Do you want to do this now, before we forget?